libmscore/sym.cpp

```diff
@@ -2683,10 +2683,12 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       QT_TRANSLATE_NOOP("symUserNames", "No symbol"),
       "4-string tab clef",
       "6-string tab clef",
-      "11 large diesis down, 3\u00b0 down [46 EDO]",
-      "11 large diesis up, (11L), (sharp less 11M), 3\u00b0 up [46 EDO]",
-      "11 medium diesis down, 1\u00b0[17 31] 2\u00b046 down, 1/4-tone down",
-      "11 medium diesis up, (11M), 1\u00b0[17 31] 2\u00b046 up, 1/4-tone up",
+      QT_TRANSLATE_NOOP("symUserNames", "11 large diesis down, 3\u00b0 down [46 EDO]"),
```

Comment:
Changing translatable strings? Not for 3.5.1 then, but up to 3.6.

Reply:
The chord playback settings PRs (just merged) already add strings too

Reply:
But 3.6 is fine by me, although that promise had been broken before already...

Reply:
Them not getting translated in time is not a big deal either

Reply:
You added it to the 3.5.1 milestones ;-) https://github.com/musescore/MuseScore/pull/6182#event-3470046745